src/components/search/index.tsx

```diff
@@ -52,9 +51,9 @@ const HitsWrapper = styled.div`
     font-size: 10px;
     position: relative;
     text-indent: -9999em;
-    border-top: 0.5em solid rgba(215,215,215, 0.2);
-    border-right: 0.5em solid rgba(215,215,215, 0.2);
-    border-bottom: 0.5em solid rgba(215,215,215, 0.2);
+    border-top: 0.5em solid rgba(215, 215, 215, 0.2);
+    border-right: 0.5em solid rgba(215, 215, 215, 0.2);
+    border-bottom: 0.5em solid rgba(215, 215, 215, 0.2);
     border-left: 0.5em solid #d7d7d7;
```

Comment:
can simplify it a bit..!
```suggestion
    border: 0.5em solid rgba(215, 215, 215, 0.2);
    border-left: 0.5em solid #d7d7d7;
```